SparkPost/2web2ui
# EO-614 Allow signals tooltips to overflow beyond chart containers

### What Changed
- Disables recharts positioning by setting `position={{ x: 0, y: 0 }}` on `<Tooltip />`
- Custom tooltip wrapper now calculates its own position

### How To Test
- Point your openresty upstreams to staging, or use `.fake` signals actions
- Visit all signals pages with charts
- Verify there are no z-index issues
- Verify tooltips switch to the left of curser if tooltip overflows past browser's right viewport edge (happens on diverging bar chart in health score)

### To Do
- [x] Maybe redo with hooks because code would look a lot cleaner
